lib/maths/CBoostedTreeFactory.cc

```diff
@@ -813,7 +801,7 @@ void CBoostedTreeFactory::initializeUnsetDownsampleFactor(core::CDataFrame& fram
                                           double minTestLoss, double testLoss) {
                     return testLoss + CTools::linearlyInterpolate(
                                           logMinDownsampleFactor, logMaxDownsampleFactor,
-                                          0.0, EPS * minTestLoss, logDownsampleFactor);
+                                          0.0, 0.01 * minTestLoss, logDownsampleFactor);
```

Comment:
This was the only place this `EPS` was used in this file. Perhaps it would be better to have it explicitly named but reference the purpose it serves. I'll come up with something better.

Reply:
I called this something more explicit in [d0be22f](https://github.com/elastic/ml-cpp/pull/1941/commits/d0be22f7f510fc797548d0f994b0ce62e5d3bedc).